Source/WebCore/platform/mediastream/gstreamer/GStreamerIncomingTrackProcessor.cpp

```diff
@@ -66,10 +66,11 @@ void GStreamerIncomingTrackProcessor::configure(ThreadSafeWeakPtr<GStreamerMedia
     auto structure = gst_caps_get_structure(m_data.caps.get(), 0);
     if (auto ssrc = gstStructureGet<unsigned>(structure, "ssrc"_s)) {
         auto msIdAttributeName = makeString("ssrc-"_s, *ssrc, "-msid"_s);
-        auto msIdAttribute = gst_structure_get_string(structure, msIdAttributeName.ascii().data());
-        auto components = String::fromUTF8(msIdAttribute).split(' ');
-        if (components.size() == 2)
-            m_sdpMsIdAndTrackId = { components[0], components[1] };
+        if (auto msIdAttribute = gstStructureGetString(structure, msIdAttributeName)) {
+            auto components = makeString(msIdAttribute).split(' ');
+            if (components.size() == 2)
+                m_sdpMsIdAndTrackId = { components[0], components[1] };
```

Comment:
You can use split with a StringView even when dealing with the result can be more complex. If you don't to deal with that complexity, you can still spare one copy if you create the string with toStringWithoutCopying .

Reply:
I find this version simpler. Dealing with a `SplitResult` here is cumbersome.

Reply:
Ok, but then create the String without copying.